features/step_definitions/networking.rb

```diff
@@ -664,15 +657,24 @@
   ensure_admin_tagged
   network_cmd = table.raw
   node_name ||= node.name
-
-  sdn_pod = BushSlicer::Pod.get_labeled("app=sdn", project: project("openshift-sdn", switch: false), user: admin) { |pod, hash|
-    pod.node_name == node_name
-  }.first
-  cache_resources sdn_pod
-  @result = sdn_pod.exec(network_cmd, as: admin)
+  _admin = admin
+   @result = _admin.cli_exec(:get, resource: "network.operator", output: "jsonpath={.items[*].spec.defaultNetwork.type}") 
+  if @result[:response] == "OpenShiftSDN"
+     sdn_pod = BushSlicer::Pod.get_labeled("app=sdn", project: project("openshift-sdn", switch: false), user: admin) { |pod, hash|
+       pod.node_name == node_name
+     }.first
+     cache_resources sdn_pod
+     @result = sdn_pod.exec(network_cmd, as: admin)
+  else
+     ovnkube_pod = BushSlicer::Pod.get_labeled("app=ovnkube-node", project: project("openshift-ovn-kubernetes", switch: false), user: admin) { |pod, hash|
```

Comment:
I do not think we have same result when run ovs command between sdn and OVN.  So do we need to change this ? we can add new step for OVN. since this will affect many version. if you intend to do this, please make sure 3.11 sdn and 4.x sdn and 4.3 OVN all works well.

Reply:
> I do not think we have same result when run ovs command between sdn and OVN. So do we need to change this

I made this step inter-operable btw ovn/sdn clusters. The existing automated test cases such as ocp-18535 will continue run for ovn as well and this is host step so any ovn test want to execute ovs cmmand on host can use this step.


> please make sure 3.11 sdn and 4.x sdn and 4.3 OVN all works well

This wont impact 3.x as 3.x definition will be fetched from v3 branch. Above code is going into master which points to 4,1+ and hence will only applicable for 4.1+ tests.

Reply:
@zhaozhanqi Can we merge this? This could will be leveraged in multus OVN testing.

Reply:
LGTM, thanks